### PR TITLE
Fireworks ai support

### PR DESCRIPTION
**Pull request by maamalama**

### CODE REVIEW

File: worker/wrangler.toml [hunk 1]
```diff
@@ -59,6 +60,7 @@ routes = [
 	{ pattern = "together.us.helicone.ai", custom_domain = true, zone_name = "helicone.ai" },
 	{ pattern = "gateway.us.helicone.ai", custom_domain = true, zone_name = "helicone.ai" },
 	{ pattern = "hyperbolic.us.helicone.ai", custom_domain = true, zone_name = "helicone.ai" },
+  { pattern = "fireworks.us.helicone.ai", custom_domain = true, zone_name = "helicone.ai" },
```
Comment (chitalian) [inline worker/wrangler.toml:63]: Nice we dont need this anymore actually
```suggestion
```

File: worker/wrangler.toml [hunk 2]
```diff
@@ -77,7 +80,7 @@ routes = [
 	{ pattern = "together.us.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
 	{ pattern = "gateway.us.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
 	{ pattern = "2yfv.us.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
-
+  { pattern = "fireworks.us.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
```
Comment (chitalian) [inline worker/wrangler.toml:83]:
```suggestion
```

File: worker/wrangler.toml [hunk 3]
```diff
@@ -86,6 +89,7 @@ routes = [
 	{ pattern = "openrouter.eu.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
 	{ pattern = "together.eu.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
 	{ pattern = "gateway.eu.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
+  { pattern = "fireworks.eu.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
```
Comment (chitalian) [inline worker/wrangler.toml:92]:
```suggestion
```

File: worker/wrangler.toml [hunk 4]
```diff
@@ -141,6 +146,7 @@ routes = [
 	{ pattern = "together.staging.eu.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
 	{ pattern = "gateway.staging.eu.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
 	{ pattern = "hyperbolic.staging.eu.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
+  { pattern = "fireworks.staging.eu.hconeai.com", custom_domain = true, zone_name = "hconeai.com" },
```
Comment (chitalian) [inline worker/wrangler.toml:149]:
```suggestion

```

**Comment by chitalian** [comment]
Can you update the docs too to include this? Or make a Linear ticket to track this?

**Comment by maamalama** [comment]
> Can you update the docs too to include this? Or make a Linear ticket to track this?

sure I gonna add docs with costs in next pr

**Pull request merged**